NickvisionTagger.Shared/Helpers/FfprobeHelpers.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text.Json;
+
+public class FfprobeFormat
+{
+    public string Filename { get; set; }
+    public Dictionary<string, string> Tags { get; set; }
+
+    public FfprobeFormat()
+    {
+        Filename = "";
+        Tags = new Dictionary<string, string>();
+    }
+    
+    public override string ToString()
+    {
+        var s = $"==={Filename}===\n";
+        foreach (var pair in Tags)
+        {
+            s += $"{pair.Key} - {pair.Value}\n";
+        }
+        s = s.Remove(s.Length - 1, 1);
+        return s;
+    }
+}
+
+public static class FfprobeHelpers
```

Comment:
Wouldn't `FFProbeHelpers` be better? Similar to code styling of MPVSharp

Reply:
Yes, I'll change it